### PR TITLE
Add Amazon_JumpStart_Semantic_Segmentation_Extract_Image.ipynb

### PR DESCRIPTION
**Pull request by ajit-aws**
*Issue #, if available:*

*Description of changes:*

*Testing done:*

## Merge Checklist

_Put an `x` in the boxes that apply. You can also fill these out after creating the PR. If you're unsure about any of them, don't hesitate to ask. We're here to help! This is simply a reminder of what we are going to look for before merging your pull request._

- [x] I have read the [CONTRIBUTING](https://github.com/aws/amazon-sagemaker-examples/blob/master/CONTRIBUTING.md) doc and adhered to the example notebook best practices
- [x] I have updated any necessary documentation, including [READMEs](https://github.com/aws/amazon-sagemaker-examples/blob/master/README.md)
- [x] I have tested my notebook(s) and ensured it runs end-to-end
- [x] I have linted my notebook(s) and code using `tox -e black-format,black-nb-format`

By submitting this pull request, I confirm that my contribution is made under the terms of the Apache 2.0 license.

### CODE REVIEW

**Comment by ajit-aws** [comment]
Cleaned up the file and updated the links

File: introduction_to_amazon_algorithms/jumpstart_semantic_segmentation/Amazon_JumpStart_Semantic_Segmentation_Extract_Image.ipynb [hunk 1]
```diff
@@ -0,0 +1,575 @@
+{
```
Comment (atqy) [inline introduction_to_amazon_algorithms/jumpstart_semantic_segmentation/Amazon_JumpStart_Semantic_Segmentation_Extract_Image.ipynb:1]: <p>Nit: "filter-out " -&gt; "filter out " </p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/aws/amazon-sagemaker-examples/pull/3410/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='9'/>

Comment (atqy) [inline introduction_to_amazon_algorithms/jumpstart_semantic_segmentation/Amazon_JumpStart_Semantic_Segmentation_Extract_Image.ipynb:1]: <p>lets -&gt; let's</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/aws/amazon-sagemaker-examples/pull/3410/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='23'/>

Comment (atqy) [inline introduction_to_amazon_algorithms/jumpstart_semantic_segmentation/Amazon_JumpStart_Semantic_Segmentation_Extract_Image.ipynb:1]: <p>Use present tense</p><p> </p><p>we will overlay-&gt; we overlay</p><p>you will find -&gt; you can find</p><p> </p><p>Also, add a comma and some grammar things (i.e. which, the)</p><p> </p><p>After executing below cell, you will find the Final_pedestrian.jpg in your folder that is extracted image of the pedestrians.  -&gt; After executing below cell, you will find the Final_pedestrian.jpg in your folder, which is the extracted image of the pedestrians.  </p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/aws/amazon-sagemaker-examples/pull/3410/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='24'/>

Comment (atqy) [inline introduction_to_amazon_algorithms/jumpstart_semantic_segmentation/Amazon_JumpStart_Semantic_Segmentation_Extract_Image.ipynb:1]: <p>Grammar things:</p><p>Lets -&gt; Let's</p><p>Original Pedestrian image with background  -&gt; original pedestrian image with the background included</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/aws/amazon-sagemaker-examples/pull/3410/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='26'/>

Comment (atqy) [inline introduction_to_amazon_algorithms/jumpstart_semantic_segmentation/Amazon_JumpStart_Semantic_Segmentation_Extract_Image.ipynb:1]: <p>Pedestrian image after extraction  -&gt; Display the pedestrian image after extraction </p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/aws/amazon-sagemaker-examples/pull/3410/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='28'/>

**Pull request merged**